### PR TITLE
Fix upgrade procedure for Debian

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Add upgrade procedure for Debian
+ fix update procedure for Debian

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [x] 22.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/update/update-centreon-platform.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Assurez-vous que tous les utilisateurs sont déconnectés avant de commencer la
 2. Mettez à jour l'ensemble des composants :
 
   ```shell
-  apt upgrade centreon\*
+  apt update centreon
```
Comment (lpinsivy) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/update/update-centreon-platform.md:68]:
```suggestion
  apt upgrade centreon
```

File: versioned_docs/version-22.10/update/update-centreon-platform.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ Please make sure all users are logged out from the Centreon web interface before
 2. Then upgrade all the components with the following command:
 
   ```shell
-  apt upgrade centreon\*
+  apt update centreon
```
Comment (lpinsivy) [inline versioned_docs/version-22.10/update/update-centreon-platform.md:67]:
```suggestion
  apt upgrade centreon
```

**Review by lpinsivy — CHANGES_REQUESTED** [review]

**Review by lpinsivy — APPROVED** [review]

**Comment by tuntoja** [comment]
@lpinsivy this line will probably fail on debian based os
`sed -ri 's/10\../10.5/' /etc/yum.repos.d/mariadb.repo`

**Comment by cg-tw** [comment]
In https://github.com/centreon/centreon-documentation/pull/1911

**Pull request closed without merging**